outlook/src/classes/Lead.ts

```diff
@@ -2,32 +2,43 @@ class Lead {
     id: number;
     name: string;
     expectedRevenue: string; // string because formatted
-    logged: boolean;
```

Comment:
Just to ask, why this removal ? I am quite new in this codebase so asking to understand a bit.

Reply:
It has been removed, because we don't verify if a lead is logged or not, furthermore, there is now a separate component handling the logging feature, so this field is no longer useful.